src/main/java/com/shipmentEvents/handlers/EventHandler.java

```diff
@@ -147,6 +147,8 @@ private List<KeyVersion> processEventsInBucket(String bucketName, LambdaLogger l
             String line1 = lines[0];
             String line2 = lines[1];
 
+            if(line1 == line2){ continue; }
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**: An equality operator (== or !=) is used to compare strings, which matches strings based on address.

**Fix**: Do content comparison using the equals() method to compare the values of the strings.